pkg/minishift/util/setproxytoshell.go

```diff
@@ -0,0 +1,32 @@
+/*
```

Comment:
@praveenkumar Looks like no such convention, only convenctions are http://stackoverflow.com/a/25162021/1120530.

But IMO, adding `_` give more readability. Like this https://github.com/minishift/minishift/blob/master/cmd/minikube/cmd/get_openshift_versions.go